themes/src/main/resources-community/theme/base/account/messages/messages_pt_BR.properties

```diff
@@ -32,33 +53,52 @@ region=Estado
 postal_code=CEP
 country=Pa\u00EDs
 emailVerified=E-mail verificado
-gssDelegationCredential=GSS Delega\u00E7\u00E3o de Credencial
-
-role_admin=Admin
-role_realm-admin=Realm Admin
-role_create-realm=Cria realm
-role_view-realm=Visualiza realm
-role_view-users=Visualiza usu\u00E1rios
-role_view-applications=Visualiza aplica\u00E7\u00F5es
-role_view-clients=Visualiza clientes
-role_view-events=Visualiza eventos
-role_view-identity-providers=Visualiza provedores de identidade
-role_manage-realm=Gerencia realm
-role_manage-users=Gerencia usu\u00E1rios
-role_manage-applications=Gerencia aplica\u00E7\u00F5es
-role_manage-identity-providers=Gerencia provedores de identidade
-role_manage-clients=Gerencia clientes
-role_manage-events=Gerencia eventos
-role_view-profile=Visualiza perfil
-role_manage-account=Gerencia conta
-role_read-token=L\u00EA token
-role_offline-access=Acesso Offline
+website=P\u00e1gina da web
+phoneNumber=N\u00famero de telefone
```

Comment:
Some of these are not part of account messages. Why they are here?

Reply:
I also couldn't find where they are used in the account theme. These lines were copied from the English file.